src/main/sensors/barometer.c

```diff
@@ -317,6 +317,11 @@ int32_t baroGetLatestAltitude(void)
     return baro.BaroAlt;
 }
 
+int16_t baroGetTemperature(void)
```

Comment:
Maybe convert to `float` and degrees? Let's start using proper SI units

Reply:
done. may you take a look why travis build fails on some targets and builds perfectly fine on most?